util/parser.py

```diff
@@ -31,7 +31,7 @@ def get_opt(main_opt, remaining_args):
         )
 
         # Save the config file when --save_config is passed
-        is_config_saved = False
+        is_config_saved = True  # False
```

Comment:
don't change this ?

Reply:
If i don not set this into True, I can not have train_config.json file, then, can not do the inference. I'll see if there has other ways

Reply:
There has no config_save option description in the joliGEN.